.github/workflows/publish-to-npm.yml

```diff
@@ -0,0 +1,26 @@
+name: npm-publish
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - main
+jobs:
+  npm-publish:
+    name: npm-publish
+    if: github.repository == 'adobe/plugin-on-fetch'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@main
+      - name: Set up Node.js
+        uses: actions/setup-node@main
+        with:
+          node-version: 18
+      - name: Publish if version has been updated
+        uses: mkiki/npm-publish-action@c4315ef5790b7bcec2cbb75b34e37681a409d78d
+        with:
+          commit_pattern: '^Release (\S+)'
+          access: "public"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```

Comment:
No i dont think we need it, i just kept it since it was there in the template https://github.com/adobe/starter-repo/blob/master/.github/workflows/npm-publish.yml. 

i have removed it as it is required only if we want to publish to github registry simultaneously with npm registry.